templates/nvidia/index.html

```diff
@@ -31,12 +31,12 @@ <h1 class="p-heading--2"><strong>Adopt AI across the enterprise with Canonical a
 		<div class="col-6">
 			<h2 class="u-sv2">Turnkey AI solutions for the enterprise powered by Canonical and NVIDIA</h2>
 			<p>Canonical and NVIDIA partner up to build turnkey solutions for enterprises in various industries to optimise their AI journey. Develop, deploy and scale AI with unparalleled hardware performance and the power of open source.</p>
-			<p><a href="https://assets.ubuntu.com/v1/ae0aedcc-AI%20enterprise%20solution%20powered%20by%20Canonical%20and%20NVIDIA-UPDATED.pdf">Check out the Enterprise-Ready AI solution powered by Canonical and NVIDIA.&nbsp;<i class="p-icon--begin-downloading"></i></a></p>
+			<p><a href="https://assets.ubuntu.com/v1/49180a1d-AI enterprise solution powered by Canonical and NVIDIA.pdf">Check out the Enterprise-Ready AI solution powered by Canonical and NVIDIA.&nbsp;<i class="p-icon--begin-downloading"></i></a></p>
```

Comment:
It's the same thing. I just encoded the spaces with `%20` the browser will do it anyways, I don't think it's good to have spaces, we could easily make a mistake and the url would be broken

Reply:
Just look at this, you can't even click on it, because it's not formatted into a link anymore with spaces... e.g. https://assets.ubuntu.com/v1/49180a1d-AI enterprise solution powered by Canonical and NVIDIA.pdf

Reply:
and yeah, we probably should use assets better than pages.ubuntu.com

Reply:
Wait, ok I think you are talking about two different things. The diff you are pointing to is the asset image, and then there is also a PDF to download. Yes they should be the same, just in different formats.